### DISCUSSION TITLE
nodes with multiple tiers

### DISCUSSION THREAD

**Discussion by jparkash2** [longhorn/longhorn#5044] [Q&A]
HI Team,

We need to deploy and test the longhorn with CAPI/CAPM3 where workload cluster having nodes with multiple tiers.
like below configuration.

Each storage nodes has HDD+SSD how to define the storage classes for HDD and as well as SSD.
How to work/integrate with multiple workload kubernates cluster using LH.

**Comment by mantissahz** [comment]
Ref: https://longhorn.io/docs/1.3.2/advanced-resources/default-disk-and-node-config/
Tag nodes or disks for usages and then create the storage class as 
Ref: https://longhorn.io/docs/1.3.2/volumes-and-nodes/storage-tags/#kubernetes

You might want a workload with a volume A on the SSD as tier 1 and a volume A on the HDD as tier 2, right?

**Reply by jparkash2** [reply]
You might want a workload with a volume A on the SSD as tier 1 and a volume A on the HDD as tier 2, right?
Yes, we need different tier for different type of workloads as well as customization of disk selection also.
e.g, If one server has 8 disk installed and we need only 6 disk should be used in longhorn and 2 should be excluded.
kindly share the configuration file to achieve such scenario to address multitiers as well as disk selection based on user configuration!

Thanks & Regards
Jai Parkash
